src/net.h

```diff
@@ -796,7 +796,8 @@ class CConnman
         nReceiveFloodSize = connOptions.nReceiveFloodSize;
         m_peer_connect_timeout = std::chrono::seconds{connOptions.m_peer_connect_timeout};
         {
-            LOCK(cs_totalBytesSent);
+            AssertLockNotHeld(m_total_bytes_sent_mutex);
+            LOCK(m_total_bytes_sent_mutex);
```

Comment:
This is not the beginning of a function. `AssertLockNotHeld()` does not belong here - it should be put in the beginning of functions.

Reply:
Fixed. Thanks.